client/allocrunner/taskrunner/stats_hook.go

```diff
@@ -22,8 +23,8 @@ type statsHook struct {
 	updater  StatsUpdater
 	interval time.Duration
 
-	// stopCh is closed by Exited or Canceled
-	stopCh chan struct{}
+	// cancel is called by Exited or Canceled
```

Comment:
Ha, I don't even remember what "Canceled" was, but it doesn't appear to exist anymore.
```suggestion
	// cancel is called by Exited
```